src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -156,29 +156,36 @@ string makeHelixWorkItemProject(AssemblyInfo assemblyInfo)
 
                 var commandLineArguments = _testExecutor.GetCommandLineArguments(assemblyInfo, useSingleQuotes: isUnix);
                 commandLineArguments = SecurityElement.Escape(commandLineArguments);
+                var setEnvironmentVariable = isUnix ? "export" : "set";
+
+                var command = new StringBuilder();
+                command.AppendLine(isUnix ? "ls" : "dir");
+                command.AppendLine(isUnix ? $"./rehydrate.sh" : $@"call .\rehydrate.cmd");
+                command.AppendLine(isUnix ? "ls" : "dir");
+                command.AppendLine($"{setEnvironmentVariable} DOTNET_ROLL_FORWARD=LatestMajor");
+                command.AppendLine($"{setEnvironmentVariable} DOTNET_ROLL_FORWARD_TO_PRERELEASE=1");
+                command.AppendLine("dotnet --info");
+
+                if (Environment.GetEnvironmentVariable("ROSLYN_TEST_IOPERATION") is string iop)
+                    command.AppendLine($"{setEnvironmentVariable} ROSLYN_TEST_IOPERATION={iop}");
+
+                command.AppendLine($"dotnet {commandLineArguments}");
+
+                // We want to collect any dumps during the post command step here; these commands are ran after the
+                // return value of the main command is captured; a failure 
+                var postCommand = new StringBuilder();
+                postCommand.AppendLine(isUnix ? "find . -name '*.dmp' -exec cp {} $HELIX_DUMP_FOLDER \\;" : "for /r %%f in (*.dmp) do copy %%f %HELIX_DUMP_FOLDER%");
```

Comment:
⚠️ It appears this command is producing an error in the following build legs:

* Test_Linux_Debug
* Test_macOS_Debug

Reply:
Yes, I still need to investigate it.